BUTTON_IDEAS.md

```diff
@@ -30,6 +31,6 @@ When adding ideas, please add them in the right category (or add a new one if yo
 - [ ] **Retro button**: A button that has a retro style to it.
 - [ ] **Neumorphic button**: A button that has a neumorphic style.
 
- ## Sign In | Sign up  Buttons
+ ## Social Buttons
 
- - [ ] **Google | FaceBook | Github** : A button with login in or sign up to Google or Facebook or Github, etc. 
\ No newline at end of file
+ - [ ] **Rounded  Buttons** : A social button with rounded corners. 
```

Comment:
Change the description to "Add rounded social buttons"

Reply:
ok